data_handler/handlers/loan_states/zklend/settings.py

```diff
@@ -22,34 +22,35 @@ class TokenSettings(ZkLendSpecificTokenSettings, TokenSettings):
     pass
 
 
-def convert_string_to_decimal_1e27(string_big_num):
+def format_reserve_data_number(string_large_num: str):
```

Comment:
Most of the newly added functionality doesn't belong into a module called `settings`. Create a new one for it 🙏🏼

Reply:
I keep the setting file as master in this commit 7a81d55a9575d342c466a44069776cadee7ab231
Also i move the fuctionality of fetch zklend settigns in this commit 73f712a53242c2402dbad84e335c8b84ae1e3ebf
NEW FILE: fetch_zklend_specific_token_settings.py